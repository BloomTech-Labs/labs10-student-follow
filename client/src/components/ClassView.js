import React from "react";
import Grid from "@material-ui/core/Grid";
import axios from 'axios';
import { withStyles } from "@material-ui/core/styles";
import { Navcrumbs, Navbar } from "./index.js";

const styles = theme => ({
  wrapper: {
    textAlign: "left"
  }
});

function ClassView(props) {
  // Variables used in multiple operations
  const headers = {
    "headers": { "Authorization": `Bearer ${process.env.REACT_APP_SENDGRID_API_KEY}` },
    "content-type": "application/json"
  }
  const listId = '7061244'

  // Basic CRUD operations
  const createList = () => {
    const url = "https://api.sendgrid.com/v3/contactdb/lists"
    const body = {
      "name": "test5"
    }
    axios.post(url, body, headers)
      .then(res => {
        console.log(`===Create list: ${res.data.name}===`)
        console.log(res.data.id)
      })
      .catch(err => console.log(err))
  }

  const getAllLists = () => {
    const url = "https://api.sendgrid.com/v3/contactdb/lists"
    axios.get(url, headers)
      .then(res => {
        console.log(`===Retrieve list: all===`)
        console.log(res.data.lists)
      })
      .catch(err => console.log(err))
  }

  const getOneList = () => {
    const url = `https://api.sendgrid.com/v3/contactdb/lists/${listId}`
    axios.get(url, headers)
      .then(res => {
        console.log(`===Retrieve list: ${res.data.name}===`)
        console.log(res.data)
      })
      .catch(err => console.log(err))
  }

  const updateOneList = () => {
    const url = `https://api.sendgrid.com/v3/contactdb/lists/${listId}`
    const body = {
      "name": "modifiedListName"
    }
    axios.patch(url, body, headers)
      .then(res => {
        console.log(`===Update list: ${res.data.name}===`)
        console.log(res)
      })
      .catch(err => console.log(err))
  }

  const deleteOneList = () => {
    const url = `https://api.sendgrid.com/v3/contactdb/lists/${listId}?delete_contacts=true`
    axios.delete(url, headers)
      .then(res => {
        console.log(`===Delete list: ${res.statusText}===`)
        console.log(res)
      })
      .catch(err => console.log(err))
  }

  return (
    <>
      <Navcrumbs />
      <Grid container spacing={0}>
        <Grid item xs={2}>
          <Navbar />
        </Grid>
        <Grid item xs={10}>
          <p>ClassView</p>
          <div>
            <button onClick={createList}>createList</button>
            <button onClick={getAllLists}>getAllLists</button>
            <button onClick={getOneList}>getOneList</button>
            <button onClick={updateOneList}>updateOneList</button>
            <button onClick={deleteOneList}>deleteOneList</button>
          </div>
        </Grid>
      </Grid>
    </>
  );
};

export default withStyles(styles)(ClassView);
