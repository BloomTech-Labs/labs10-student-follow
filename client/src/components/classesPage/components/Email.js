import React from 'react';
import ReactDOMServer from 'react-dom/server';

function makeEmail(refreshr) {
  return ReactDOMServer.renderToStaticMarkup(
    <html>
      <head>
        <title />
      </head>
      <body>
        <p>
          Take your Refreshr <a href={refreshr.typeform_url}>here</a>.
        </p>
        <p />
        <a href="[unsubscribe]">Unsubscribe</a>
      </body>
    </html>
  );
}

export default makeEmail;
