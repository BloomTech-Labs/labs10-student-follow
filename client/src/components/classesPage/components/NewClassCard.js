import React from "react";
import Grid from "@material-ui/core/Grid";
import { withStyles } from "@material-ui/core/styles";
import { Link } from 'react-router-dom'

const styles = theme => ({
  wrapper: {
    border: "1px solid black",
    display: "flex",
    flexWrap: "wrap",
    alignItems: "center",
    justifyContent: "center",
    margin: "1rem",
    width: "30%",
    height: "200px"
  }
});

const NewClassCard = props => {
  return (
    <Grid className={props.classes.wrapper}>
      <Link to="/classes/create">
        <h1>New Class</h1>
      </Link>
    </Grid>
  );
};
export default withStyles(styles)(NewClassCard);
