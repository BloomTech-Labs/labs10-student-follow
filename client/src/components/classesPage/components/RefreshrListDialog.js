import React from 'react';
import { withStyles } from '@material-ui/core/styles';
import { Link } from 'react-router-dom';
import {
  List,
  ListItem,
  ListItemText,
  DialogTitle,
  Dialog
} from '@material-ui/core/';
import AddCircleOutline from '@material-ui/icons/AddCircleOutline';

const styles = theme => ({
  dialog: {
    border: `1px solid ${theme.palette.secondary.main}`
  },
  listItem: {
    color: theme.palette.secondary.main
  },
  title: {
    color: theme.palette.secondary.main
  },
  link: {
    textDecoration: 'none'
  },
  icon: {
    color: theme.palette.secondary.main
  }
});

const RefreshrDialog = props => {
  const { classes } = props;
  return (
    <div>
      <Dialog
        classes={{ paper: classes.dialog }}
        onClose={props.handleClose}
        open={props.open}
      >
        <DialogTitle>
          <span className={classes.title}>Select a refreshr</span>
        </DialogTitle>
        <div>
          <List>
            {props.refreshrs.map(r => (
              <ListItem
                button
                onClick={() => props.selectNewRefreshr(r.refreshr_id)}
                key={r.refreshr_id}
              >
                <ListItemText
                  classes={{ primary: classes.listItem }}
                  primary={r.name}
                />
              </ListItem>
            ))}
            <Link className={classes.link} to="/refreshrs/create">
              <ListItem>
                <AddCircleOutline className={classes.icon} />
                <ListItemText
                  classes={{ primary: classes.listItem }}
                  primary={'Create a new refreshr'}
                />
              </ListItem>
            </Link>
          </List>
        </div>
      </Dialog>
    </div>
  );
};

export default withStyles(styles)(RefreshrDialog);
