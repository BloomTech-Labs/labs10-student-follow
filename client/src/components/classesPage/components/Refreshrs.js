import React from 'react';
import {
  Grid,
  Typography,
  Card,
  CardContent,
  Button,
  TextField,
  List,
  ListItem,
  ListItemIcon,
  ListItemText,
  ListSubheader,
  Fab
} from '@material-ui/core';
import { withStyles } from '@material-ui/core/styles';
import RefreshrDialog from './RefreshrListDialog';
import {
  RemoveCircleOutline,
  Update,
  AddCircleOutline
} from '@material-ui/icons/';
import moment from 'moment';
import logo from './LogoSmall.png';

const styles = theme => ({
  refreshrCardWrapper: {
    display: 'flex',
    padding: theme.spacing.unit,
    [theme.breakpoints.only('xs')]: {
      flexFlow: 'column nowrap',
      justifyContent: 'space-between',
      alignItems: 'center'
    },
    justifyContent: 'center'
  },
  refreshrCard: {
    border: `1px solid ${theme.palette.secondary.main}`,
    color: `${theme.palette.primary.contrastText}`,
    margin: theme.spacing.unit,
    display: 'flex',
    flexFlow: 'column nowrap',
    alignItems: 'center',
    fontSize: '1rem',
    minWidth: 375,
    minHeight: 225,
    justifyContent: 'center',
    [theme.breakpoints.only('xs')]: {
      width: '100%'
    }
  },
  refreshrContent: {
    fontSize: '1.0rem',
    display: 'flex',
    flexDirection: 'column',
    alignItems: 'center'
  },
  refreshrTitle: {
    color: `${theme.palette.primary.contrastText}`
  },
  refreshrList: {
    border: `1px solid ${theme.palette.secondary.main}`,
    color: `${theme.palette.primary.contrastText}`,
    margin: theme.spacing.unit,
    display: 'flex',
    flexFlow: 'column nowrap',
    borderRadius: '5px',
    backgroundColor: theme.palette.primary.main
  },

  refreshrIcon: {
    alignSelf: 'flex-end',
    margin: '5%',
    '&:hover': {
      cursor: 'pointer'
    }
  },
  datePicker: {
    color: theme.palette.primary.contrastText,
    fontSize: '1.1rem'
  },
  title: {
    color: `${theme.palette.primary.contrastText}`,
    textAlign: 'center'
  },
  button: {
    marginRight: theme.spacing.unit * 2,
    marginTop: theme.spacing.unit * 2,
    color: theme.palette.primary.main,
    background: theme.palette.secondary.main,
    width: 40,
    height: 40
  },
  listItem: {
    fontSize: '1.1rem',
    '&:hover': {
      backgroundColor: '#363c42'
    },
    '&:focus': {
      backgroundColor: '#363c42'
    }
  },
  logo: {
    width: 25,
    height: 25
  },
  subhead: {
    color: theme.palette.primary.contrastText
  },
  listIcon: {
    color: theme.palette.primary.contrastText
  },
  listItemText: {
    fontSize: '1.1rem'
  }
});

function Refreshrs(props) {
  const { classes } = props;
  function setDate(e) {
    props.setAddedRefreshr({ ...props.addedRefreshr, date: e.target.value });
  }
  function closeModal() {
    props.setModalIsOpen(false);
  }
  return (
    <>
      <Grid>
        <Typography variant="h6" className={classes.title} gutterBottom>
          Refreshrs
        </Typography>
        <Grid className={classes.refreshrCardWrapper}>
          {props.activeRefreshr ? (
            <Card className={classes.refreshrCard} raised>
              <CardContent className={classes.refreshrContent}>
                <Typography
                  variant="subtitle2"
                  className={classes.refreshrTitle}
                >
                  {props.activeRefreshr.name}
                </Typography>
              </CardContent>
              <CardContent className={classes.refreshrContent}>
                <form
                  styles={{ display: 'flex', flexFlow: 'column' }}
                  onSubmit={e => props.submitNewDate(e)}
                >
                  <TextField
                    variant="outlined"
                    type="date"
                    InputProps={{ className: classes.datePicker }}
                    value={moment(props.activeDate).format('YYYY-MM-DD')}
                    onChange={e => props.changeDate(e)}
                  />
                </form>
                <Fab
                  className={classes.button}
                  onClick={props.submitNewDate}
                  elevation={20}
                  aria-label="Update"
                >
                  <Update />
                </Fab>
              </CardContent>
              <RemoveCircleOutline
                onClick={() =>
                  props.removeRefreshr(props.activeRefreshr.refreshr_id)
                }
                className={classes.refreshrIcon}
              />
            </Card>
          ) : props.addedRefreshr ? (
            <Card className={classes.refreshrCard} raised>
              <CardContent className={classes.refreshrContent}>
                <Typography
                  variant="subtitle2"
                  className={classes.refreshrTitle}
                >
                  {props.addedRefreshr.name}
                </Typography>
              </CardContent>
              <CardContent className={classes.refreshrContent}>
                <TextField
                  onChange={e => setDate(e)}
                  variant="outlined"
                  type="date"
                  InputProps={{ className: classes.datePicker }}
                  defaultValue={moment().format('YYYY-MM-DD')}
                />
              </CardContent>
              {props.addedRefreshr.date && (
                <Button onClick={props.addRefreshr}>Submit</Button>
              )}
            </Card>
          ) : (
            <Card className={classes.refreshrCard} raised>
              <CardContent className={classes.refreshrContent}>
                <Typography
                  variant="subtitle2"
                  className={classes.refreshrTitle}
                >
                  Select a Refreshr To Edit
                </Typography>
              </CardContent>
            </Card>
          )}
        </Grid>
        <List
          component="ul"
          className={classes.refreshrList}
          subheader={
            <ListSubheader className={classes.subhead} component="div">
              {props.className} Refreshrs
            </ListSubheader>
          }
        >
          {props.refreshrs.map(r => (
            <ListItem
              className={classes.listItem}
              button
              key={r.refreshr_id}
              onClick={() => props.selectRefreshr(r.refreshr_id)}
            >
              <ListItemIcon>
                <img src={logo} alt="refreshr logo" className={classes.logo} />
              </ListItemIcon>

              <ListItemText
                classes={{ primary: classes.listItemText }}
                primary={r.name}
              />
            </ListItem>
          ))}
          <ListItem
            button
            onClick={() => props.setModalIsOpen(!props.modalIsOpen)}
          >
            <ListItemIcon>
              <AddCircleOutline className={classes.listIcon} />
            </ListItemIcon>
            <ListItemText
              classes={{ primary: classes.listItemText }}
              primary="Add new refreshr to class"
            />
          </ListItem>
        </List>
        <RefreshrDialog
          refreshrs={props.teacherRefs}
          open={props.modalIsOpen}
          handleClose={closeModal}
          selectNewRefreshr={props.selectNewRefreshr}
        />
      </Grid>
    </>
  );
}

export default withStyles(styles, { withTheme: true })(Refreshrs);
