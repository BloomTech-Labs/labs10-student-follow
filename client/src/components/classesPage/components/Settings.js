import React from 'react';
import { Typography, Fab } from '@material-ui/core';
import { Create } from '@material-ui/icons';
import { withStyles } from '@material-ui/core/styles';

import Update from '@material-ui/icons/Update';

const styles = theme => ({
  nameForm: {
    // border: `1px solid ${theme.palette.secondary.main}`,
    display: 'flex',
    flexFlow: 'column nowrap',
    justifyContent: 'space-around',
    alignItems: 'center',
    padding: '5%'
  },
  inputBtnDiv: {
    border: '1px solid red',
    display: 'flex',
    flexFlow: 'column nowrap',
    paddingLeft: '10%'
  },
  btn: {
    marginRight: theme.spacing.unit * 2,
    marginTop: theme.spacing.unit * 2,
    color: theme.palette.primary.main,
    background: theme.palette.secondary.main,
    width: 40,
    height: 40
  }
});

function Settings(props) {
  const { classes } = props;
  return (
    <>
      <Typography
        variant="h6"
        color="secondary"
        style={{ textAlign: 'center' }}
      >
        Settings
      </Typography>
      {props.isEditingClass ? (
        <form
          className={classes.nameForm}
          onSubmit={e => props.changeClassName(e)}
        >
          <Typography variant="body1" gutterBottom>
            Edit Classname
          </Typography>
          {props.isEditingClass &&
            props.makeInput(
              'className',
              'Class Name',
              props.classData.name,
              e => {
                props.handleClassChange(e);
              }
            )}
          <Fab
            elevation={20}
            aria-label="Update"
            className={classes.btn}
            onClick={e => props.changeClassName(e)}
          >
            <Update />
          </Fab>
        </form>
      ) : (
        <>
          <Typography variant="body1" gutterBottom>
            {props.classData.name}
          </Typography>
          <Create onClick={() => props.setIsEditingClass(true)} />
        </>
      )}
    </>
  );
}

export default withStyles(styles, { withTheme: true })(Settings);
