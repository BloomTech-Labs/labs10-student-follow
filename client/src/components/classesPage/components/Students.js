import React from 'react';
import { withStyles } from '@material-ui/core/styles';
import {
  Grid,
  Typography,
  Card,
  Button,
  Fab,
  ExpansionPanel,
  ExpansionPanelSummary
} from '@material-ui/core';
import {
  GroupAdd,
  ExpandMore,
  Create,
  Backspace,
  RemoveCircleOutline
} from '@material-ui/icons';

const styles = theme => ({
  studentList: {
    display: 'flex',
    flexFlow: 'column wrap',
    border: `1px solid ${theme.palette.secondary.main}`,
    flexWrap: 'wrap',
    width: '50%',
    [theme.breakpoints.down('sm')]: {
      width: '70%'
    },
    maxHeight: theme.spacing.unit * 50,
    padding: theme.spacing.unit * 2
  },
  settingsBox: {
    display: 'flex',
    flexFlow: 'column nowrap',
    justifyContent: 'space-around',
    alignItems: 'center',
    padding: '5%',
    [theme.breakpoints.down('sm')]: {
      flexDirection: 'column',
      alignItems: 'center'
    },
    //border: '1px solid purple',
    margin: theme.spacing.unit * 2
  },
  title: {
    color: `${theme.palette.primary.contrastText}`,
    textAlign: 'center'
  },
  btn: {
    marginRight: theme.spacing.unit * 2,
    marginTop: theme.spacing.unit * 2,
    color: theme.palette.primary.main,
    background: theme.palette.secondary.main,
    width: 40,
    height: 40
  },
  hiddenButton: {
    display: 'none'
  },
  expansionPanel: {
    marginTop: theme.spacing.unit * 3,
    borderRadius: '5px',
    border: `1px solid ${theme.palette.secondary.main}`,
    width: '50%',
    [theme.breakpoints.down('sm')]: {
      width: '70%'
    }
  },
  editName: {
    display: 'flex',
    flexDirection: 'column'
  },
  studentName: {
    display: 'flex',
    justifyContent: 'space-between',
    marginBottom: theme.spacing.unit * 2,
    alignItems: 'center'
  },
  studentIcons: {
    display: 'inline',
    alignSelf: 'flex-end'
  }
});

function students(props) {
  const { classes } = props;

  return (
    <>
      <Typography variant="h6" className={classes.title} gutterBottom>
        Current Students
      </Typography>
      <Card className={classes.studentList}>
        {props.students.map(s => (
          <Grid
            key={s.student_id}
            className={s.isActiveStudent ? null : classes.studentName}
          >
            <span>{`${s.first_name} ${s.last_name} `}</span>
            <Grid className={classes.studentIcons}>
              {s.isActiveStudent ? (
                <Backspace onClick={e => props.toggleEditStudent(s)} />
              ) : (
                <Create onClick={e => props.toggleEditStudent(s)} />
              )}
              <RemoveCircleOutline
                onClick={() => props.dropStudent(s.student_id)}
              />
            </Grid>
            {s.isActiveStudent && (
              <form
                className={classes.editName}
                onSubmit={e => props.submitUpdatedStudent(e)}
              >
                {props.makeInput(
                  'email',
                  'Email',
                  props.activeStudent.email,
                  e => {
                    props.updateStudent(e, s);
                  }
                )}
                {props.makeInput(
                  'first_name',
                  'First Name',
                  props.activeStudent.first_name,
                  e => {
                    props.updateStudent(e, s);
                  }
                )}
                {props.makeInput(
                  'last_name',
                  'Last Name',
                  props.activeStudent.last_name,
                  e => {
                    props.updateStudent(e, s);
                  }
                )}
                <button className={classes.hiddenButton} />
              </form>
            )}
          </Grid>
        ))}
      </Card>
      <ExpansionPanel className={classes.expansionPanel}>
        <ExpansionPanelSummary expandIcon={<ExpandMore />}>
          <Typography variant="body1" gutterBottom>
            Add a Student
          </Typography>
        </ExpansionPanelSummary>
        <form
          className={classes.settingsBox}
          onSubmit={e => props.addStudent(e)}
        >
          {props.makeInput('email', 'Email', undefined, undefined, 'email')}
          {props.makeInput('first_name', 'First Name')}
          {props.makeInput('last_name', 'Last Name')}
          <Fab
            elevation={20}
            aria-label="Add"
            className={classes.btn}
            type="submit"
            onClick={e => props.addStudent(e)}
          >
            <GroupAdd />
          </Fab>
        </form>
      </ExpansionPanel>
      <Grid className={classes.buttonBox}>
        {props.selectedStudents.length ? (
          <Button variant="outlined" onClick={props.dropStudents}>
            Remove selected from class
          </Button>
        ) : null}
      </Grid>
    </>
  );
}

export default withStyles(styles, { withTheme: true })(students);
