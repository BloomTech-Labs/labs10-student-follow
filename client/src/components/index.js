/* COMMON */
import Navcrumbs from './navigation/Navcrumbs';
import Navbar from './navigation/Navbar';

/* PAGES */
import BillingPage from './billingPage/BillingPage';
import LandingPage from './landingPage/LandingPage';
import RefreshrsPage from './refreshrsPage/RefreshrsPage';

/* BILLING */
import TakeMoney from './billingPage/TakeMoney';
import PricingModal from './billingPage/PricingModal'

/* CLASSES */
import ClassCreateView from './classesPage/ClassCreateView';
import ClassEditView from './classesPage/ClassEditView';
import ClassListView from './classesPage/ClassListView';
import CampaignForm from './classesPage/forms/CampaignForm';
import ListForm from './classesPage/forms/ListForm';

/* REFRESHRS */
import RefreshrListView from './refreshrsPage/RefreshrListView';
import RefreshrCard from './refreshrsPage/RefreshrCard';
import Refreshr from './refreshrsPage/Refreshr';
import RefreshrEdit from './refreshrsPage/RefreshrEdit';

/* MISC DATA */
import SendgridTestOps from './classesPage/SendgridTestOps';
import Dashboard from './dashboardPage/Dashboard';

export {
  Navcrumbs,
  Navbar,
  BillingPage,
  LandingPage,
  RefreshrsPage,
  TakeMoney,
  ClassCreateView,
  ClassEditView,
  ClassListView,
  CampaignForm,
  ListForm,
  RefreshrListView,
  RefreshrCard,
  Refreshr,
  RefreshrEdit,
  SendgridTestOps,
  Dashboard,
  PricingModal
};
