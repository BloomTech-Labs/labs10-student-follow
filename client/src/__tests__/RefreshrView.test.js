import React from "react";
import { render, cleanup } from "react-testing-library";
import "jest-dom/extend-expect";
import { RefreshrView } from "../components";

afterEach(cleanup);

const wrapper = {
  wrapper: {
    textAlign: "left"
  }
};

describe("default test", () => {
  it("renders correctly", () => {
    const { container } = render(<RefreshrView className={wrapper} />);
    expect(container).toMatchSnapshot();
  });
});
