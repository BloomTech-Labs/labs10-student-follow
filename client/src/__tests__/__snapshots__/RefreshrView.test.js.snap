// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`default test renders correctly 1`] = `
<div>
  <div
    class="Navcrumbs-wrapper-2"
  >
    <p
      data-testid="crumbTrail"
    >
      Home &gt; Refreshrs
    </p>
  </div>
  <div
    class="MuiGrid-container-3"
  >
    <div
      class="MuiGrid-item-4 MuiGrid-grid-xs-2-33"
    >
      <div
        class="Navbar-wrapper-100"
      >
        <p
          data-testid="firstMenuItem"
        >
          Refreshrs
        </p>
        <p
          data-testid="secondMenuItem"
        >
          Classes
        </p>
        <p
          data-testid="thirdMenuItem"
        >
          Billing
        </p>
        <p
          data-testid="fourthMenuItem"
        >
          Settings
        </p>
      </div>
    </div>
    <div
      class="MuiGrid-item-4 MuiGrid-grid-xs-10-41"
    >
      <div
        class="RefreshrList-wrapper-101"
      >
        <div
          class="RefreshrList-wrapper-102"
        >
          <h1>
            Card
          </h1>
        </div>
        <div
          class="RefreshrList-wrapper-102"
        >
          <h1>
            Card
          </h1>
        </div>
        <div
          class="RefreshrList-wrapper-102"
        >
          <h1>
            Card
          </h1>
        </div>
      </div>
    </div>
  </div>
</div>
`;
